### PR TITLE
Register the commands and deploy the code on CFW

### PR DESCRIPTION
**Pull request by RitikJaiswal75**
Discord provides us with an endpoint to register all our commands to our bots.
The register.ts file runs on our pipeline and hist that endpoint with all the commands that we want to register.

Created a few files to register all the commands given in commands.ts to the bot.

Added a pipeline to deploy code to register the commands and deploy code to the Cloudflare workers for the staging environment.

For this pipeline to work we must set up the environment variables in the GitHub environment.

### CODE REVIEW

File: src/utils/registerCommands.ts [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+    headers: {
+      "Content-Type": "application/json",
+      Authorization: `Bot ${discordBotToken}`,
+    },
+    method: "PUT",
+    body: JSON.stringify(commands),
+  });
+
+  if (response.ok) console.log("Registered all commands");
+  else console.error("Error Registering Commands");
```
Comment (harshith-venkatesh) [inline src/utils/registerCommands.ts:19]: Would it be good to handle async calls via try/catch

**Review by harshith-venkatesh — DISMISSED** [review]
Everything looks good @RitikJaiswal75 , added a comment.
I believe all the tokens are added and github actions are proof tested.
Good work

**Comment by RitikJaiswal75** [comment]
> Everything looks good @RitikJaiswal75 , added a comment. I believe all the tokens are added and github actions are proof tested. Good work

Yes Harshith, I reviewed them over a POC

File: src/utils/registerCommands.ts (hunk 1 above)
Comment (RitikJaiswal75) [reply src/utils/registerCommands.ts]: Updated the PR

**Review by harshith-venkatesh — APPROVED** [review]

**Comment by prakashchoudhary07** [comment]
> > Everything looks good @RitikJaiswal75 , added a comment. I believe all the tokens are added and github actions are proof tested. Good work
> 
> Yes Harshith, I reviewed them over a POC
> 
> I am still waiting for the variables to be added to the GitHub environment by @prakashchoudhary07

Have added the env secrets

**Review by lakshayman — APPROVED** [review]

**Review by prakashchoudhary07 — APPROVED** [review]
LGTM 👍

**Pull request merged**